### ISSUE TITLE
Handle ties/playoff holes

### ISSUE THREAD

**Issue by ericgio** [ericgio/weekday-golf#37]
We go to a playoff hole if there's a tie on the last hole (and maybe if the round is tied?) but the results aren't reflected in the data. There are also other tiebreaker methods that we haven't utilized, like looking at the last N holes. That kind of approach could be used going forward to break ties, though it's not nearly as fun as a playoff.

**Comment by makinde** [comment]
If we go with a playoff, I can handle this with some data fields/storage stuff I'm doing. lmk what you prefer that or something else. I like the playoff. Recency aside, I'd vote that everyone in the skins get back into the sudden death putoff since I think it aligns better with how the rest of the skins competition works.

**Comment by ericgio** [comment]
Yeah I'm a fan of the putt-off for sure. More complex to codify, though

**Comment by makinde** [comment]
I'm planning to add a field to the round, `skinPlayoffWinner`
